docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  derbynet:
+    ports:
+      - "4443:443"
```

Comment:
I'm admittedly still new to the docker world, and haven't yet had occasion to use docker compose, but I'm not sure why these are the right ports.  

Do you envision being able to start the image with just a docker-compose up?  Otherwise I don't really understand why have a single-service docker compose file.

Reply:
Yes.  Docker compose up is easier to grok than the full docker command, it also allows for potential integrations with different database solutions besides using the sqlite file.  You could then add a section for postgres, mysql or whatever and link the containers for testing. The strange port numbers where just to avoid root privileges for ports <1024 on unix style operating systems.